answers/Maker.md

```diff
@@ -0,0 +1,179 @@
+
+For governance votes, which are non-binding and used to poll MKR holder sentiment about larger issues, votes can be time-limited. If the vote is time-limited, votes are tallied at the end of the voting period and a simple majority (<50%) determines the outcome.
+
+Executive voting is not time-limited, but instead employs continuous [approval voting](https://en.wikipedia.org/wiki/Approval_voting). Whichever proposal currently has the most votes represents the current state of the system. There is no quorum, incentivizing MKR holders' continuous participation. At any time, a new proposal can be submitted to MKR holders (e.g. a proposal to lower the debt ceiling to decrease exposure to ETH). If it gains a majority of votes, it will be automatically implemented. New proposals are not immediately implemented however. There is a 24 hr delay period, in which "Emergency Oracles" can trigger an emergency shutdown in the event of "long-term market irrationality", hacking, or security breaches. In multi-collateral DAI, Emergency Oracles will have the unilateral ability to trigger an emergency shutdown. This will give a minority of MKR holders the ability to trigger a shutdown if they believe the "governance community" (core team) is making off-chain decisions that have become biased or corrupted. For example, if large MKR holders try to get favorable risk parameters for an asset they hold to pump their bags.
+
+Changes to existing risk parameters (variables in existing smart contracts) can be implemented automatically. Major upgrades involving changes to smart contract logic must be performed through the emergency shutdown process (i.e. rebooting the entire system).
+
+## External actors
+
+- **Price oracles:** To estimate the price of a collateral asset, Maker uses a number of (hopefully) independent price oracles. To protect against the possibility of a price oracle becoming compromised, the median of all price oracle prices is used as input to the system. To protect against the unlikely scenario that enough oracles are compromised to move the median price, a "Oracle Security Module" delays the price fed into the system by one hour. If the Oracle Security Module detects an attack, it can issue an emergency shutdown. The Oracle Security Module is comprised of members of the core team and Emergency Oracles selected by MKR holders.
+- **Keepers:** Keepers are external, permissionless actors that participate in the Debt Auctions and Collateral Auctions when CDPs are liquidated. They can also perform arbitrage when the price of DAI diverges from its $1 peg, or other profit-seeking activities.
+- **Emergency oracles:** Emergency oracles are selected by MKR holders and have the ability to unilaterally trigger an emergency shutdown. They serve as a last line of defense, should a malicious proposal or governance attack threaten the system.
```

Comment:
Interesting, are they named individuals? Selected by MKR vote? Would love to see a link to one of these "elections" and some more detail.

Reply:
Rune Christensen's in a recent Unchained podcast [interview](https://unchainedpodcast.com/rune-christensen-of-makerdao-part-1-how-to-keep-a-crypto-collateralized-stablecoin-afloat/) was talking iirc about oracles having been selected from early, trusted community members and organizations. Their identities aren't public for opsec reasons. In the long-run, I believe the plan is that these oracles are trusted institutions that could be sued in their respective jurisdiction should they perform a "governance" or "price oracle" attack. Could probably dig up a reddit thread laying this out if we need (they do pretty much everything on Reddit).